src/families/cosmos/js-prepareTransaction.ts

```diff
@@ -121,9 +121,10 @@ const prepareTransaction = async (
   if (
     transaction.memo !== memo ||
     transaction.fees !== fees ||
-    transaction.gas !== gas
+    transaction.gas !== gas ||
+    transaction.amount !== amount
```

Comment:
correct me if i'm wrong but I think amount is a big number so the `!==` should be `! .eq()` no?